gitlab/tests/objects/test_project_merge_request_approvals.py

```diff
@@ -230,6 +233,17 @@ def resp_snippet():
         yield rsps
 
 
+def test_project_approval_manager_update_uses_post(project, resp_snippet):
+    """Ensure the
+    gitlab.v4.objects.merge_request_approvals.ProjectApprovalManager object has
+    _update_uses_post set to True"""
```

Comment:
I was originally thinking this would be more appropriate in a separate test/fake manager, to test just this specific aspect of it, e.g. in `test_object_mixin_attributes.py`. But since it's only used here I think it's ok, maybe for a follow-up PR :)

Reply:
I can do that, though I'm not quite sure what you want me to do 😕

Do you want a test that shows that `update_uses_post` exists and is False in `test_object_mixin_attributes.py` ?